### PR TITLE
Update movies

### PR DESCRIPTION
**Pull request by rvargas**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -474,7 +474,74 @@ <h2 id="movie-title"></h2>
             releaseDate: '2016-05-27',
             releaseDateReadable: 'May 27, 2016',
             thumbnailUrl: 'https://upload.wikimedia.org/wikipedia/en/1/12/The_Do-Over_Poster.png',
-            ratings: []
+            ratings: [
+              {
+                name: 'IMDb',
+                value: '4.9/10',
```
Comment (thewarpaint) [inline index.html]: Isn't it 5.7? http://www.imdb.com/title/tt4769836/

File: index.html [hunk 2]
```diff
@@ -474,7 +474,74 @@ <h2 id="movie-title"></h2>
...
+                link: 'http://www.imdb.com/title/tt5893332/'
+              },
+              {
+                name: 'Rotten Tomatoes',
+                value: '32%',
+                link: 'https://www.rottentomatoes.com/m/sandy_wexler/#contentReviews'
+              },
+              {
+                name: 'Metacritic',
+                value: '22%',
```
Comment (thewarpaint) [inline index.html]: I see 40%: http://www.metacritic.com/movie/sandy-wexler

File: index.html [hunk 3]
```diff
@@ -474,7 +474,74 @@ <h2 id="movie-title"></h2>
...
+                link: 'http://www.metacritic.com/movie/sandy-wexler'
+              }
+            ]
+          },
+          {
+            title: 'The Meyerowitz Stories (New and Selected)',
```
Comment (thewarpaint) [inline index.html:524]: That's kinda against the idea, but I'll allow it.

**Review by thewarpaint — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (rvargas) [reply index.html:524]: :1st_place_medal:

**Review by thewarpaint — APPROVED** [review]

**Pull request merged**